src/PSDocs.Azure/docs/Azure.Template.Doc.ps1

```diff
@@ -236,5 +236,16 @@ Document 'README' {
                 $example | Code 'json'
             }
         }
+        # Add command line snippet
+        if ($PSDocs.Configuration.GetBoolOrDefault('AZURE_USE_COMMAND_LINE_SNIPPET', $True)) {
+            Section $LocalizedData.CommandLine {
```

Comment:
@vicperdana I think the build errors are mostly related to this.

You need to add an entry to the `src/PSDocs.Azure/en/PSDocs-strings.psd1` file for `CommandLine` which is the localized string.

Reply:
Thanks @BernieWhite - have added this.